HED-schema-reduced.mediawiki

```diff
@@ -340,29 +350,55 @@ There are four ways to define the scope of an event.  The most common method is
 ** Urban
 ** Rural
 
-* Cognitive modifier <nowiki>{extensionAllowed}</nowiki>
-** Imagined <nowiki>[This is used to identity that the (sub)event only happened in the imagination of the participant, e.g.  imagined movements in motor imagery paradigms.]</nowiki>
-** Reflexive
-** Intentional
+* Experimental <nowiki>{extensionAllowed}</nowiki>
 ** Incidental
```

Comment:
Can it be used anywhere else besides with Action?

Reply:
I agree it isn't quite there --- but I don't know how to fix.  Do you have any specific ideas?  If not, I may go ahead and merge so we can look again in the expanded view.  We need to a catch-all category.